docs/core/preview3/tools/csproj.md

```diff
@@ -25,81 +25,103 @@ see [the MSBuild project file](https://docs.microsoft.com/visualstudio/msbuild/m
 be consumed.
 
-The element can contain one or more of the following values:
+The attribute can contain one or more of the following values:
 
-* Compile – the contents of the lib folder are available to compile against.
-* Runtime – the contents of the runtime folder are distributed.
-* ContentFiles – the contents of the contentfiles folder are used.
-* Build – the props/targets in the build folder are used.
-* Native – the contents from native assets are copied to the output folder for runtime.
-* Analyzers – the analyzers are used.
+* `Compile` – the contents of the lib folder are available to compile against.
+* `Runtime` – the contents of the runtime folder are distributed.
+* `ContentFiles` – the contents of the contentfiles folder are used.
+* `Build` – the props/targets in the build folder are used.
+* `Native` – the contents from native assets are copied to the output folder for runtime.
+* `Analyzers` – the analyzers are used.
 
 Alternatively, the element can contain:
 
-* None – none of the assets are used.
-* All – all assets are used.
+* `None` – none of the assets are used.
+* `All` – all assets are used.
 
 #### PrivateAssets
-`<PrivateAssets>` child element specifies what assets belonging to the package specified by parent `<PackageReference>` should be 
+`<PrivateAssets>` attribute specifies what assets belonging to the package specified by `<PackageReference>` should be 
 consumed but that they should not flow to the next project. 
 
 > [!NOTE]
 > This is a new term for project.json/xproj `SuppressParent` element. 
 
-The element can contain one or more of the following values:
+The attribute can contain one or more of the following values:
 
-* Compile – the contents of the lib folder are available to compile against.
-* Runtime – the contents of the runtime folder are distributed.
-* ContentFiles – the contents of the contentfiles folder are used.
-* Build – the props/targets in the build folder are used.
-* Native – the contents from native assets are copied to the output folder for runtime.
-* Analyzers – the analyzers are used.
+* `Compile` – the contents of the lib folder are available to compile against.
+* `Runtime` – the contents of the runtime folder are distributed.
+* `ContentFiles` – the contents of the contentfiles folder are used.
+* `Build` – the props/targets in the build folder are used.
+* `Native` – the contents from native assets are copied to the output folder for runtime.
+* `Analyzers` – the analyzers are used.
 
-Alternatively, the element can contain:
+Alternatively, the attribute can contain:
 
-* None – none of the assets are used.
-* All – all assets are used.
+* `None` – none of the assets are used.
+* `All` – all assets are used.
 
 ### DotnetCliToolReference
 `<DotnetCliToolReference>` element specifies the CLI tool that the user wants restores in the context of the project. It is 
```

Comment:
`DotnetCliToolReference` specifies...

that the user wants restores --> that the user wants to restore?, to restore?